ods_ci/tasks/Resources/Provisioning/GPU/amd_operator/amd_operator.sh

```diff
@@ -56,35 +56,44 @@ has_csv_succeeded() {
 }
 
 function create_devconfig() {
-  oc create -f - <<EOF
+  dc_name="dc-internal-registry"
+  dc=$(oc get DeviceConfig $dc_name -n openshift-amd-gpu -oname --ignore-not-found)
+  if [[ -n $dc ]];
+    then
+      echo "AMD DeviceConfig $dc_name already exists". Skipping creation
+    else
+      echo "Creating AMD DeviceConfig..."
+      oc create -f - <<EOF
 kind: DeviceConfig
 apiVersion: amd.io/v1alpha1
 metadata:
-  name: dc-internal-registry
+  name: $dc_name
   namespace: openshift-amd-gpu
 EOF
+  fi
 }
 
-function wait_until_pod_ready_status() {
-  local timeout_seconds=1200
-  local pod_label=$1
-  local namespace=$2
-  local timeout=240
+
+function wait_until_pod_is_created() {
+  label=$1
+  namespace=$2
+  timeout=$3
   start_time=$(date +%s)
   while [ $(($(date +%s) - start_time)) -lt $timeout ]; do
-     pod_status="$(oc get pod -l app="$pod_label" -n "$namespace" --no-headers=true 2>/dev/null)"
-     daemon_status="$(oc get daemonset -l app="$pod_label" -n "$namespace" --no-headers=true 2>/dev/null)"
-     if [[ -n "$daemon_status" || -n "$pod_status" ]] ; then
-        echo "Waiting until GPU Pods or Daemonset of '$pod_label' in namespace '$namespace' are in running state..."
-        echo "Pods status: '$pod_status'"
-        echo "Daemonset status: '$daemon_status'"
-        oc wait --timeout="${timeout_seconds}s" --for=condition=ready pod -n "$namespace" -l app="$pod_label" || \
-        oc rollout status --watch --timeout=3m daemonset -n "$namespace" -l app="$pod_label" || continue
+    podName=$(oc get pods -n $2 -l $1 -oname)
+    if [[ -n $podName ]];
+      then {
+        echo Pod $podName found!
+        return 0
         break
-     fi
-     echo "Waiting for Pods or Daemonset with label app='$pod_label' in namespace '$namespace' to be present..."
-     sleep 5
+      } else {
+        echo "waiting for pod with label $label"
+        sleep 1
+      }
+    fi
   done
+  echo "Timeout exceeded, pod with label $label not found"
```

Comment:
maybe it will be good to add kind of this message before we start the `while` loop

Reply:
why? this msg must appaer only after the timeout is exceeded, hence after the loop

Reply:
Sorry, I was not clear :) 
what I meant is to add a message like:
`  echo "Start waiting for pod with label $label for $timeout seconds"`
in this way, the user will know what the expected timeout from the logs

Reply:
I thought it would be redudant with `echo "waiting for pod with label $label"`

Reply:
I agree I could have added the timeout information. Do you mind if I push this change another time? I've tested this PR and the corresponding one in Jenkins multiple times, would like to merge them

Reply:
Sure NP